newIDE/app/src/InstructionOrExpression/EventsScope.js

```diff
@@ -1,5 +1,9 @@
 // @flow
-const gd: libGDevelop = global.gd;
+
+// The way `gd` is usually used cannot be used because `ProjectScopedContainersAccessor`
+// is used in `makeTestProject` right after GDevelopJS initialization. Doing so makes
+// the storybook crash.
+// const gd: libGDevelop = global.gd;
```

Comment:
Super weird. Can you still use `const gd: libGDevelop = global.gd` but inside the functions using it?

Reply:
Good idea, I'll do this